end-to-end-untargeted-metabolomics.Rmd

```diff
@@ -349,6 +365,8 @@ finite value (default is `Inf`) which will cause the data to be processed (and
 loaded into memory) in chunks of `processingChunkSize()` spectra. This can
 reduce memory demand and speed up the process.
 
+[Dan: Does the bin function also uses the maximum intensity of the binned spectra, or does it averages it? Also, at the plots below there`s a RT information as a heading fore each plot. Are those plots the BPS of the whole data or the BPC of RT=0.3?]
+
```

Comment:
I believe it the default parameter of the `bin()` function takes the maximum yes :) 
For the title i have a doubt, i think this  might be a bug I will ask johannes !